src/spanner_grpc_config.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
So is this all essentially fine tuning for each API endpoint we access?  Who is supposed to keep this up to date?

Reply:
Currently it's configured as this to address the max 100 concurrent streams issue, but I think as for who's responsible for future changes in this file, it depends on which parts to change. For example if the affinity key changes, then the service owner needs to change it, and if the grpc channel max throughput changes, then we should make the changes accordingly. Does that sound reasonable?

Reply:
My one thought here is if there is a way to combine this with our other configuration at some point. We already specify retry conditions for rpc calls.

Reply:
@crwilcox We already maintain a general ApiConfig proto definition for all languages, so I believe it would be good to have a separate config file specifically for this proto. Also FWIK in future more grpc features (like retry logic and regional support) will also be supported in this grpc-gcp library. In that case those grpc related configs can all be put into this config file.. WDYT?